myst_nbs/howto/howto_debugging.myst.md

```diff
@@ -6,108 +6,184 @@ jupytext:
     format_version: 0.13
     jupytext_version: 1.13.7
 kernelspec:
-  display_name: Python 3
+  display_name: Python 3 (ipykernel)
   language: python
   name: python3
 ---
 
+(howto_debugging)=
 # How to debug a model
 
-There are various levels on which to debug a model. One of the simplest is to just print out the values that different variables are taking on.
+:::{post} July 11, 2016
+:tags: 
+:category: beginner
+:author: Thomas Wiecki
```

Comment:
Both you and Thomas should be listed here. From the "first cell" section:

> Authors should list people who authored, adapted or updated the notebook. See [Authorship and attribution](https://docs.pymc.io/en/latest/contributing/jupyter_style.html#jupyter-authors) for more details.

and from the authorship section:

> updated
>
>    for notebooks that have not only been re-executed but have also had significant updates to their content (either code, explanations or both).

which I believe is what applies here.